lib/classes/tooltip.js

```diff
@@ -1,11 +1,10 @@
 import Popper from 'popper.js';
 import { assign, keys } from '../utils/object';
 import { isArray, from as arrayFrom } from '../utils/array';
-import { assign, keys } from '../utils/object';
 
 const NAME = 'v-b-tooltp';
 const CLASS_PREFIX = 'bs-tooltip';
-const BSCLS_PREFIX_REGEX = new RegExp(`(^|\\s)${CLASS_PREFIX}\\S+`, 'g');
+const BSCLS_PREFIX_REGEX = new RegExp(`\\b${CLASS_PREFIX}\\S+`, 'g');
```

Comment:
See https://github.com/twbs/bootstrap/blob/v4-dev/js/src/tooltip.js#L37

Reply:
it matches with space at start

Reply:
why to blindly follow things, that could be done better?

Reply:
https://regex101.com/r/Kocwnl/1

Reply:
I know theirs works. Just trying to keep as close to their code (although changing all jQuery to native DOM functions), as it will make it easier to include any fixes that TWBS makes (they just made a fwe commit changes to Tooltip/Popover today)

Reply:
and this is BS's 
https://regex101.com/r/OgnXoS/1
String.match returns full match

Reply:
I got almost whole this PR fixed... one thing left - it does not respect position

Reply:
I suggest to do a PR to BS repo :)

Reply:
jQuery allows removing a bunch of class in a single `$(el).remoceClass(...)` operation (space separated), whereas native DOM `el.classList.remove(...)` only allows one class name (no spaces) to be removed per call... that is why they were capturing the space before the class name (there can be multiple classes with the prefix on the tooltip element)